### DISCUSSION TITLE
5.4.0.1 (Hotfix)

### DISCUSSION THREAD

**Discussion by T-Troll** [T-Troll/alienfx-tools#98] [Announcements]
Low-level SDK:
- Bug fixed: After BIOS update (at least for m15R3-R4), status codes changed, so device status retrieve incorrectly for APIv5.

`AlienFX-GUI`:
- Bug fixed: Slow application start for APIv5 devices (see above)
- Bug fixed: Light testing stop working (slow update and colors not changed) for APIv5 devices (see above)
- Bug fixed: At "Device" tab, unselect light into lights list now update state and information correctly.
- Bug fixed: Screen flicker at the app start if "Run minimized" enabled.

<hr /><em>This discussion was created from the release <a href='https://github.com/T-Troll/alienfx-tools/releases/tag/5.4.0.1'>5.4.0.1 (Hotfix)</a>.</em>

**Comment by ghost** [comment]
Nice Software.
Thanks for saving me from the headache that is AWCC

**Reply by T-Troll** [reply]
Answering your first question (i see you found yourself already) - select it, then press Enter or double-click.
Yes, i need better documentation...

PS: I will appreciate if you share light ID's/names for your gear (use "Save lights" button) - i'll add it into database for auto-detection.
